tsconfig.json

```diff
@@ -16,6 +16,7 @@
         "noImplicitAny": false,
         "strictBindCallApply": false,
         "forceConsistentCasingInFileNames": false,
-        "noFallthroughCasesInSwitch": false
+        "noFallthroughCasesInSwitch": false,
+        "resolveJsonModule": true
```

Comment:
can you explain please why you add this one?

Reply:
It is because I need to import  json drafts from the json schema package